### ISSUE TITLE
Multiple job schedules for one command.

### ISSUE THREAD

**Issue by AndreiArdelean1** [alseambusher/crontab-ui#134]
It would be nice to be able to specify multiple times for the same command. In the UI, the job can have an array of schedules, and when deploying to create multiple cron jobs.

For example: 
{command: "echo Hello", schedules: ["@reboot", "@weekly", "0 0 3 * 5"]}
should generate 3 cron jobs.

**Comment by tuxuday5** [comment]
Can't list option be used for your scenario?

**Comment by AndreiArdelean1** [comment]
Yes it can, but it would be cleaner to have them groped. If you need to modify one command, you would have to modify all the commands. Also, if you are logging the output, it you'd be distributed across multiple files.

**Comment by tuxuday5** [comment]
Normally its recommended to group all @startup jobs in a script and schedule it. Same goes for @yearly & @monthly.

In real scenarios we come across jobs that needs to be run every weekday @5AM and @5PM, this can be done using list.

Please elobarate your use case.
